deploy.sh

```diff
@@ -14,7 +14,7 @@ sentry-cli releases -o $SENTRY_ORG new -p $SENTRY_PROJECT $RELEASE
 sentry-cli releases -o $SENTRY_ORG -p $SENTRY_PROJECT set-commits --auto $RELEASE
 sentry-cli releases -o $SENTRY_ORG -p $SENTRY_PROJECT files $RELEASE upload-sourcemaps --url-prefix "~/static/js" --validate build/$PREFIX
 
-# The release was set in the static prod build
+# This deploys React - The release was set in the static prod build
 gcloud app deploy --quiet
 
-cd ../flask && gcloud app deploy --quiet
+# cd ../flask && gcloud app deploy --quiet
```

Comment:
Is there a reason the flask deploy is commented out here?

Reply:
this was a mistake, from when I was testing. But just now I added a react/deploy.sh so I can use that instead, and not have to bother editing this deploy.sh here